app/Main.hs

```diff
@@ -92,7 +95,7 @@ rules env = do
         indexContext =
           listField "tutorials" tutorialCtx (return tutorials)
             <> constField "title" "Home"
-            <> constField "title-list" "Tutorials"
+            <> constField "title-list" ""
```

Comment:
I think an extra space will be added anyway, right?

Reply:
Yes, but the best way is add a conditional in template, because if is a space, it will create a `h1` empty tag (isn't good for SEO). I will fix it.

Reply:
Forget it, Hakyll doesn't allow comparisons in conditionals: https://jaspervdj.be/hakyll/tutorials/04-compilers.html#conditionals